databricks/koalas/series.py

```diff
@@ -4730,6 +4730,30 @@ def asof(self, where):
         result_series.name = self.name
         return result_series
 
+    def item(self):
+        """
+        Return the first element of the underlying data as a Python scalar.
+
+        Returns
+        -------
+        scalar
+            The first element of Series.
+
+        Raises
+        ------
+        ValueError
+            If the data is not length-1.
+
+        Examples
+        --------
+        >>> kser = ks.Series([10])
+        >>> kser.item()
+        10
+        """
+        if len(self[:2]) != 1:
```

Comment:
Can you reuse the output from `self[:2]` rather then executing the jobs twice?

Reply:
Ah, I missed it. Thanks. !